mpsl/doc/cx.rst

```diff
@@ -30,13 +30,13 @@ This experimental feature has several limitations:
 Implementation
 **************
 
-To use the radio coexistence feature, you can either select the implementation available in NCS or replace it with another implementation compatible with your PTA.
+To use the radio coexistence feature, you can either select the implementation available in nRF Connect SDK or replace it with another implementation compatible with your PTA.
 Each implementation exposes the same API to allow for an easier replacement that does not require modifications to the radio protocols.
 
-The radio coexistence implementation provided in NCS is the following:
+The radio coexistence implementation provided in RF Connect SDK is the following:
 
 * *Thread Radio Coexistence*.
-  This PTA interface is compatible with the 3-wire PTA interface defined in the *Thread Radio Coexistence recommendations* document.
```

Comment:
@hubertmis , are you ok with these changes?

Reply:
To explain, I took out the reference because, to my knowledge, the document is non-public. In that case, referencing it seems odd.

Reply:
Thread protocol specification has the same confidentiality level (it's available only for Thread Group members), but we mention it in Thread protocol specification. I don't think it's good to avoid mentioning which specification was used as a base of implementation, only because not everyone has access to the spec.

Reply:
Would an acceptable alternative then be:
"This PTA interface is compatible with the 3-wire PTA interface defined in *Thread Radio coexistence recommendations*."

This removes reference to the document, but keeps the source specification.

Reply:
I think it's acceptable, but it makes this sentence ambiguous.

Let's say a Thread Group member reads cx.rst documentation. They understand "Thread Radio Coexistence" implementation is compatible with some _Thread Radio coexistence recommendations_. But what are these recommendations? Did documentation author mean the *Thread Radio Coexistence recommendations* document? Any other document? Or recommendations that were discussed by one of the working groups?

I would keep the original sentence, because it was clear.

Reply:
Alright, I reverted the sentence to its original form, though I changed the preceding line from a bullet point to a term definition.

I still think it is an odd reference in the context that the reader does not already know of the existence of that specific document. For example, googling the document name gave me absolutely no related results. If the reader is not already familiar, they have no way of knowing where to gain more information. I gues it's a case of assuming a certain level of familiarity with Thread for anyone who reads the `cx.rst`.

Reply:
We can clarify for users unfamiliar with this document, that it is available for Thread Group members, like a footnote or something

Reply:
I can provide a link to the document. It leads to Thread Group portal and only registered users have access

Reply:
Can the Thread Group be credited as an author for the document? That gives an indication of where to find more information. So "[...} document by the Thread Group."